neo/IO/Caching/Cache.cs

```diff
@@ -92,7 +92,7 @@ private void AddInternal(TKey key, TValue item)
                 if (InnerDictionary.Count >= max_capacity)
                 {
                     //TODO: Perform a performance test on the PLINQ query to determine which algorithm is better here (parallel or not)
-                    foreach (CacheItem item_del in InnerDictionary.Values.AsParallel().OrderBy(p => p.Time).Take(InnerDictionary.Count - max_capacity + 1))
+                    foreach (CacheItem item_del in InnerDictionary.Values.OrderBy(p => p.Time).Take(InnerDictionary.Count - max_capacity + 1))
```

Comment:
Why this change?

Reply:
1 second to few ms

Reply:
@shargon  but this is in neo code,  is this change intended?

Reply:
I can revert it if you want, but at least with small collections i tested that is worst in parallel. I think that we should be removed the parallel until take the decision of this TODO.